Update .vimrc.bundles

andreimerlescu opened the pull request:
Removed repo that prompted for Github authentication.

I have installed this package on hundreds of systems and this has annoyed me one last time. Regardless if this PR is accepted, please hit the thumbs up button on this if you too have been annoyed by the prompt asking you to sign into Github in order to install this amazing package.

andreimerlescu committed:
Update .vimrc.bundles

Removed repo that prompted for Github authentication.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/spf13/spf13-vim?pullRequest=1129) <br/>All committers have signed the CLA.

andreimerlescu committed:
Update README.markdown

andreimerlescu committed:
Update bootstrap.sh

andreimerlescu committed:
Update bootstrap.sh

Set the URI to https:// from git:// to allow un-configured SSH users to access the installation of this unbroken spf-13 vim installer.
